time stamp

int32 id
int64 count
int64 count_changed
int64 freq
int64 freq_change
int8 dlc
int8[] data
string[] variables
bool active

